i18n/locales/en/common.json

```diff
@@ -245,7 +245,7 @@
   "Lisk Terms of Use": "Lisk Terms of Use",
   "Lisk Website": "Lisk Website",
   "Lisk accounts on {{WalletModel}}": "Lisk accounts on {{WalletModel}}",
-  "Lisk counts your message in bytes, so keep in mind\n                that the length of your message may vary in different languages.\n                Different characters may consume a varying amount of bytes.": "Lisk counts your message in bytes, so keep in mind\n                that the length of your message may vary in different languages.\n                Different characters may consume a varying amount of bytes.",
+  "Lisk counts your message in bytes, so keep in mind\r\n                that the length of your message may vary in different languages.\r\n                Different characters may consume a varying amount of bytes.": "Lisk counts your message in bytes, so keep in mind\r\n                that the length of your message may vary in different languages.\r\n                Different characters may consume a varying amount of bytes.",
```

Comment:
Why are these returns introduced in locales?

Reply:
It's from the IDE or it happens when I start the app locally. Either way, I'll remove them